modules/oe_whitelabel_search/oe_whitelabel_search.module

```diff
@@ -2,7 +2,7 @@
 
 /**
  * @file
- * OE Whitelabel Search Module.
+ * Module file used for theming the search feature.
```

Comment:
For my taste we should agree on a copyable comment we can put in every *.module file. The only party interested in these `@file` docs is phpcs..
But this is not a blocker, ok to leave it as is.